src/App.js

```diff
@@ -191,25 +190,21 @@ class App extends Component {
         <section id="main" className="container">
           {this.state.articles
             .filter(article => {
-              if (this.state.selectedSource === "All") {
-                return article;
-              } else if (article.source === this.state.selectedSource) {
-                return article;
-              }
-              return null;
+              return this.state.selectedSource === "All"
```

Comment:
the argument you pass to filter only needs to return true or false, (not `article` or `null`) so you can simplify this filter to a one liner as shown.